deck.gl/examples/pure-js/angular/carto2/src/app/modules/home/layers/rail-roads-layer.ts

```diff
@@ -11,16 +11,17 @@ export class RailRoadsLayer extends Layer {
   visible = true;
 
   async getLayer() {
-    return new CartoSQLLayer({
+    return new CartoLayer({
       id: this.id,
+      type: MAP_TYPES.QUERY,
       data: 'SELECT cartodb_id, the_geom_webmercator, category, scalerank FROM ne_10m_railroads_public',
       visible: this.visible,
       pickable: true,
       lineWidthScale: 20,
       lineWidthMinPixels: 2,
       getLineColor: colorContinuous({
         attr: 'scalerank',
-        domain: [1, 2, 3, 4, 5, 10],
+        domain: [4, 5, 6, 7, 8, 9, 10],
```

Comment:
ok, if you think it's better 👍🏻

Reply:
Yes, there is no scalerank smaller than 4